wallet/src/wallet.rs

```diff
@@ -117,6 +117,15 @@ impl<B: storage::Backend> Wallet<B> {
     }
 }
 
+pub fn generate_new_mnemonic() -> bip39::Mnemonic {
+    let mut rng = crypto::random::make_true_rng();
+    let mut data = [0u8; 32];
+    rng.fill(&mut data);
+    let res = bip39::Mnemonic::from_entropy(&data).expect("should not fail");
```

Comment:
Please add a TODO for the language. We can support other languages for free.

Reply:
Well, frankly, it's not that hard to do. You can add a parameter to this function for language, then just pass English... then we do the CLI implementation for that later.